src/arrays/ArrayUtil.java

```diff
@@ -47,14 +94,40 @@ public static boolean contains(int[] array, int value) {
      */
     public static int indexOf(int[] array, int value) {
 
-        return 0;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                return i;
+            }
+        }
+
+        System.out.println();
+
+        return -1;
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
+        int[] newarr0 = new int[array.length - 1];
```

Comment:
you can do it by using just one for, in the for just do not add the old element with the given index